requirements-base.txt

```diff
@@ -66,7 +66,7 @@ toronado==0.1.0
 typing-extensions==3.10.0.2
 ua-parser==0.10.0
 unidiff==0.7.1
-urllib3[brotli]==1.25.11
+urllib3[brotli]==1.26.8
 brotlipy==0.7.0
```

Comment:
urllib changed the brotli library -- you'll want to make sure you're using the right one (or just remove the `brotlipy` pin)

Reply:
Good catch, I was just relying on their changelog (which doesn't document https://github.com/urllib3/urllib3/issues/2574) and didn't actually check.